Equipment/EquipmentDb/Weapons/1hSwords.xml

```diff
@@ -40,6 +40,31 @@
 		</source>
 	</melee_weapon>
 
+	<melee_weapon id="13246" phase="1">
+		<info name="Argent Avenger" type="SWORD" slot="1H" unique="no" req_lvl="55" item_lvl="62" quality="RARE" boe="no" icon="Inv_sword_05.png"/>
+		<dmg_range min="71" max="108" speed="2.2"/>
+		<proc>
+			<spell name="ATTACK_POWER_UNDEAD" min="200" max="200" instant="yes" rate="0.041666677" internal_cd="0" duration="10"/>
```

Comment:
The proc is not created for this item as the spell ATTACK_POWER_UNDEAD doesn't exist, and instead of implementing it you can change this to a buff proc. You can find an example in 2hSwords.xml, The Untamed Blade has a Strength proc.

Reply:
Does the buff proc ATTACK_POWER_UNDEAD exist, though?

Reply:
> Does the buff proc ATTACK_POWER_UNDEAD exist, though?

It didn't so I added it just now in a317cb7d60ea91efdfd98b387915ec297d4a347d so you can rebase if you want to try it out locally.